server/notebooks/migrations/0002_notebook_forked_from.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1 on 2018-10-03 01:54
```

Comment:
@wlach I should probably remove this, yeah? There is another migration added to the repository so I added.

Reply:
Fine to leave the comment in.